presto-accumulo/src/main/java/com/facebook/presto/accumulo/conf/AccumuloTableProperties.java

```diff
@@ -0,0 +1,247 @@
+                false, false);
+
+        PropertyMetadata<String> s4 = stringSessionProperty(LOCALITY_GROUPS,
+                "List of locality groups to set on the Accumulo table. Only valid on internal tables. "
+                        + "String format is locality group name, colon, comma delimited list of Presto "
+                        + "column names in the group. Groups are delimited by pipes. Example: "
+                        + "group1:colA,colB,colC|group2:colD,colE,colF|etc.... Default is no locality groups.",
+                null, false);
+
+        PropertyMetadata<String> s5 = stringSessionProperty(ROW_ID,
+                "Presto column name that maps to the Accumulo row ID. Default is the first column.",
+                null, false);
+
+        PropertyMetadata<String> s6 =
+                new PropertyMetadata<>(SERIALIZER,
+                        "Serializer for Accumulo data encodings. Can either be 'default', "
+                                + "'string', 'lexicoder', or a Java class name. Default is 'default', i.e. "
+                                + "the value from AccumuloRowSerializer.getDefault(), i.e. 'lexicoder'.",
+                        VarcharType.VARCHAR, String.class,
+                        AccumuloRowSerializer.getDefault().getClass().getName(), false,
+                        x -> x.equals("default")
+                                ? AccumuloRowSerializer.getDefault().getClass().getName()
+                                : (x.equals("string") ? StringRowSerializer.class.getName()
+                                : (x.equals("lexicoder")
+                                ? LexicoderRowSerializer.class.getName()
+                                : (String) x)),
+                        object -> object);
+
+        PropertyMetadata<String> s7 = stringSessionProperty(SCAN_AUTHS,
+                "Scan-time authorizations set on the batch scanner. Default is all scan authorizations for the user",
+                null, false);
+
+        tableProperties = ImmutableList.of(s1, s2, s3, s4, s5, s6, s7);
+    }
+
+    /**
+     * Gets all available table properties
+     *
+     * @return A List of table properties
+     */
+    public List<PropertyMetadata<?>> getTableProperties()
+    {
+        return tableProperties;
+    }
+
+    /**
+     * Gets the value of the column_mapping property and parses it into a map of Presto column name
+     * to a pair of strings, the Accumulo column family and qualifier.
+     *
+     * @param tableProperties The map of table properties
+     * @return The column mapping, presto name to (accumulo column family, qualifier)
+     */
+    public static Map<String, Pair<String, String>> getColumnMapping(
+            Map<String, Object> tableProperties)
+    {
+        String strMapping = (String) tableProperties.get(COLUMN_MAPPING);
+        if (strMapping == null) {
+            return null;
```

Comment:
I'd prefer to use `Optional` here to align with (what will soon be) the rest of the table properties.